src/LeanCode.ContractsGenerator/ProjectLoader.cs

```diff
@@ -79,27 +79,37 @@ private static void VerifyProject(IProjectAnalyzer project)
             }
 
             var proj = workspace.CurrentSolution.GetProject(id);
-            if (proj is null)
+            if (proj is not null)
```

Comment:
Inversion of this conditional seems unnecessary; I think early `throw`s and `return`s are more readable over something like

```csharp
if (thing is not null) // or (thing is Type expected)
{
    …

    …wall of code…

    …
}
else
{
    throw …;
}
```

And we have one less level of necessary indentation as a bonus.

Reply:
I, unfortunately, think otherwise. Early returns/throws, apart from argument validation, tend to break the code flow and are harder to maintain IMO.

Reply:
Plus, I really dislike `if`s without `else` :)